treeherder/webapp/api/jobs.py

```diff
@@ -354,7 +354,7 @@ def list(self, request, project):
                     # no jobs are excluded for this project, we should return
                     # an empty array and skip the query altogether.
```

Comment:
Could you update the comment to say "an empty queryset" instead of "an empty array"?